content/posts/2016/pytest-development-sprint.md

```diff
@@ -0,0 +1,87 @@
+
+Discussions:
+
+* what to include in pytest 3.0
+* advanced intro sessions on plugin mechanics (pluggy) and the fixture system implementation
+* tox roadmap
+* [Started to conceptualize generalized package builds and virtualenv creation](https://bitbucket.org/hpk42/tox/issues/338/generalize-package-builds-and-virtualenv)
+* process for removing deprecated features
+* proposal for [combining parametrize with fixtures](https://github.com/pytest-dev/pytest/pull/1660)
+* new 'fromcontext'/'invocation' scope for fixtures and some initial prototyping  
+* lightning talks
+
+![Post it notes for planning]({attach}images/sprint_postits.jpg){.img-rounded}
+
+Major features:
+
+* dropping pytest assert reinterpret code
+* [rename binary from 'py.test' to 'pytest'](https://github.com/pytest-dev/pytest/issues/1629) (but don't worry, ``py.test`` will still work)
+* [documentation restructuring](https://github.com/pytest-dev/pytest/wiki/Docs-refactor)
+* [``--setup-only`` and ``--setup-plan`` flags which show how fixtures are initialised without actually running the tests](https://github.com/pytest-dev/pytest/pull/1647)
+* Adding ``-o`` commandline option to override .ini config values
+
+Other features:
+
+* pytest-qt features - improvements towards a [2.0 release](https://github.com/pytest-dev/pytest-qt/blob/master/CHANGELOG.rst)
+* pytest-bdd enhancements
+* pytest-django: triaged issues in general and made some progress especially with regard to better handle DB access/setup.
+* pytest-html enhancements
+* pytest-factoryboy enhancements
+* pytest-selenium enhancements
+* pytest-repeat enhancements
+* pytest-environment research
+* cookiecutter-pytest-plugin improvements
+* 4 projects [submitted](http://pytest.org/latest/contributing.html#submitting-plugins-to-pytest-dev) to pytest-dev organisation
+
+Bugs fixed/in progress:
+
+* [exit pytest on collection error](https://github.com/pytest-dev/pytest/issues/1421)
+* [markers stain on all related classes](https://github.com/pytest-dev/pytest/issues/568)
+* [removed deprecated command line options](https://github.com/pytest-dev/pytest/issues/1657)
+* [incorrectly dropping brackets on display of assertions](https://github.com/pytest-dev/pytest/issues/925)
+* [rename getfuncargvalue to getfixturevalue](https://github.com/pytest-dev/pytest/issues/1625)
+* [warning if you use getfuncargvalue with parametrized fixtures](https://github.com/pytest-dev/pytest/issues/460)
+* [terminal newlines in failed test output](https://github.com/pytest-dev/pytest/issues/1553)
+* [escaping curly braces in a tox command doesn't work](https://bitbucket.org/hpk42/tox/issues/212)
+* [Tox shouldn't call pip directly to avoid shebang limitations](https://bitbucket.org/hpk42/tox/issues/66)
+* [missing evaluated value in report when asserting a boolean attribute](https://github.com/pytest-dev/pytest/issues/1503)
+* [improved determination of rootdir](https://github.com/pytest-dev/pytest/pull/1621)
+
+
```

Comment:
While you said that this is work started/finished on the sprint, perhaps it is worth to emphasize that this is not a complete list of features that will go into 3.0:

> It is worth noting that this is not an exhaustive list of all features that will go into pytest-3.0; a separate post after the release will be made with all major highlights.

What do you think?